### PR TITLE
Feature database

### PR DESCRIPTION
**Pull request by gustavoleite**
Added data manipulation with the firebase. The app auto-logs through the anonymous session. Documents can be saved, edited, and deleted.

### CODE REVIEW

File: src/actions/DocumentActions.js [hunk 1]
```diff
@@ -13,15 +17,53 @@ export const documentUpdate = ({ prop, value }) => {
...
 
 export const documentCreate = ({ name, number, image }) => {
-  //todo
+  const { currentUser } = firebase.auth();
+  return (dispatch) => {
+    firebase.database().ref(`/users/${currentUser.uid}/documents`)
+      .push({ name, number, image })
+      .then(() => {
+        dispatch({ type: DOCUMENT_CREATE });
+        //Actions.employeeList({ type: 'reset' });
```
Comment (Morbix) [inline src/actions/DocumentActions.js]: Remove comments or add TODO mark

File: src/actions/DocumentActions.js [hunk 2]
```diff
@@ -13,15 +17,53 @@ export const documentUpdate = ({ prop, value }) => {
...
+};
+
+export const documentFetch = () => {
+  const { currentUser } = firebase.auth();
+  console.log(currentUser);
+  return (dispatch) => {
+    firebase.database().ref(`/users/${currentUser.uid}/documents`)
+      .on('value', snapshot => {
+        dispatch({ type: DOCUMENT_FETCH, payload: snapshot.val() });
+          console.log(snapshot.val());
```
Comment (Morbix) [inline src/actions/DocumentActions.js]: Remove logs.

Comment (gustavoleite) [reply src/actions/DocumentActions.js]: Ok

File: src/actions/DocumentActions.js (hunk 1 above)
Comment (gustavoleite) [reply src/actions/DocumentActions.js]: Ok

**Review by Morbix — APPROVED** [review]

**Pull request merged**